CHANGES

```diff
@@ -1,3 +1,9 @@
+v1.4.1 - 2016-02-08
```

Comment:
Hello!
I was about to create a fork for this issue but I saw this open PR.
Both v.1.4.1 & v.1.4.0 report the release year as 2016 when it should be 2017.
Thanks 😊